notification/twilio/voice.go

```diff
@@ -572,10 +581,12 @@ func (v *Voice) ServeAlert(w http.ResponseWriter, req *http.Request) {
 	if call == nil {
 		return
 	}
+	cfg := config.FromContext(ctx)
 
 	// See Twilio Request Parameter documentation at
 	// https://www.twilio.com/docs/api/twiml/twilio_request#synchronous
 	resp := newTwiMLResponse(w)
```

Comment:
Let's add `ctx` as an arg to `newTwiMLResponse` and have it read in the config internally so we don't need to track calling `AddVoiceOptions` at every call site

Reply:
adjusted as per your request